.github/workflows/js-ci.yml

```
@@ -194,18 +178,18 @@ jobs:
           KEYCLOAK_ADMIN_PASSWORD: admin
 
       - name: Install Playwright browsers
-        run: pnpm --filter ${{ env.WORKSPACE }} exec playwright install --with-deps
+        run: pnpm --fail-if-no-match --filter ${{ env.WORKSPACE }} exec playwright install --with-deps
 
       - name: Run Playwright tests
-        run: pnpm --filter ${{ env.WORKSPACE }} test
+        run: pnpm --fail-if-no-match --filter ${{ env.WORKSPACE }} test
         env:
           KEYCLOAK_SERVER: http://localhost:8080
 
       - uses: actions/upload-artifact@v4
         if: always()
         with:
-          name: account-ui-playwright-report
-          path: js/apps/${{ env.WORKSPACE }}/playwright-report
+          name: account-ui-traces
+          path: js/apps/account-ui/test-results
           retention-days: 30
```

I think we should follow the recommendations from the [Playwright documentation](https://playwright.dev/docs/ci-intro) on GitHub Actions here and keep uploading the full report in the `playwright-report` directory, which can then be viewed [using the command line](https://playwright.dev/docs/ci-intro#viewing-the-html-report).

```suggestion
          name: account-ui-playwright-report
          path: js/apps/account-ui/playwright-report
          retention-days: 30
```